### ISSUE TITLE
The peptide-binding residue prediction in CAMP_train_CV.py

### ISSUE THREAD

**Issue by a30221274** [twopin/CAMP#44]
Hi,

I trained The Binary Interaction Prediction task using my own data in CAMP_train_CV.py. Could you please provide the code for training The Peptide-Binding Residue Prediction task? 

Thank you very much.

**Comment by twopin** [comment]
Hi, for the BS prediction I only have the keras version (I didn't reimplement the pytorch version). But I think I posted the keras version of revising the CAMP with multi-objective loss. You can just revise the loss in the CAMP-torch version to get a Binding site prediction model. If you have any questions, you can email me at leiyp21@mails.tsinghua.edu.cn, which is my current email. I think I'm considering re-implement the bs prediction model by torch since some people raise the same requirement recently.

**Comment by a30221274** [comment]
Thank you very much for your reply. 
In addition, may I ask you for a python jupyterbook to make inference as you shared in #39 problem unpickling CAMP_pytorch files.
